README.md

```diff
@@ -1,2 +1,282 @@
+- OpenShift CLI is installed.
+- Operator Lifecycle Manager (OLM) has been installed in your cluster.
+- Your cluster has a default storage class provisioned.
+- Helm v3.9+ is installed.
+
+Note that as of November 6, 2023, OpenShift Serverless Operator is based on RHEL 8 images which are not supported on the ARM64 architecture. Consequently, deployment of this helm chart on an OpenShift Local cluster on Macbook laptops with M1/M2 chips is not supported.
+
+## Installation
+
+This helm chart deploys OpenShift Serverless Operator and SonataFlow operator as well as other components using custom resources (eg, KnativeEventing, KnativeServing CRs, SonataFlow, SonataFlowPlatform, etc) for the CRDs supported by these operators. Because the custom resources can only be deployed after the corresponding CRDs for the operator have been created and registered with the Kubernetes API Server, we use a two-phase deployment process as follows.
+
+Build helm dependency and create a new project for the installation:
+```console
+$ git clone git@github.com:parodos-dev/orchestrator-helm-chart.git
+$ cd orchestrator-helm-chart/charts
+$ helm dep update orchestrator
+$ oc new-project orchestrator-install
+```
+Perform a first pass install:
+```console
+$ helm install orchestrator orchestrator
+NAME: orchestrator
+LAST DEPLOYED: Tue Nov  7 21:04:36 2023
+NAMESPACE: orchestrator-install
+STATUS: deployed
+REVISION: 1
+NOTES:
+Helm Release orchestrator installed in namespace orchestrator-install.
+
+Components                   Installed   Namespace
+====================================================================
+Backstage                    YES        orchestrator-install
+Postgres DB - Backstage      YES        orchestrator-install
+Red Hat Serverless Operator  YES        openshift-serverless
+KnativeServing               NO         knative-serving
+KnativeEventing              NO         knative-eventing
+SonataFlow Operator          YES        openshift-operators
+SonataFlowPlatform           NO         sonataflow-infra
+Data Index Service           YES        sonataflow-infra
+Job Service                  YES        sonataflow-infra
+Postgres DB - workflows      YES        sonataflow-infra
+
+No workflows deployed.
+
+Run the following commands to wait until the services are ready:
+  oc wait -n openshift-serverless deploy/knative-openshift --for=condition=Available --timeout=5m
+  oc wait -n openshift-operators deploy/sonataflow-operator-controller-manager --for=condition=Available --timeout=5m
+  oc wait -n sonataflow-infra deploy/data-index --for=condition=Available --timeout=5m
+  oc wait -n sonataflow-infra deploy/jobs-service --for=condition=Available --timeout=5m
+  oc wait -n sonataflow-infra pod/postgres-db-0 --for=condition=Ready --timeout=5m
+  oc wait -n orchestrator-install pod/orchestrator-postgresql-0 --for=condition=Ready --timeout=5m
+  oc wait -n orchestrator-install deploy/orchestrator-backstage --for=condition=Available --timeout=5m
```

Comment:
Are you sure about this namespace?
```bash
$ oc wait -n orchestrator-install pod/orchestrator-postgresql-0 --for=condition=Ready --timeout=5m
Error from server (NotFound): namespaces "orchestrator-install" not found
```
It seems that the pods are on the default namespace:

```bash
$ oc get pods
NAME                                      READY   STATUS             RESTARTS      AGE
orchestrator-backstage-59cb9f49df-dmhwx   0/1     CrashLoopBackOff   4 (50s ago)   3m12s
orchestrator-postgresql-0 
```

Reply:
ok i see that there is the command `oc new-project orchestrator-install`

Reply:
OK